VAGRANT.md

```diff
@@ -44,3 +47,7 @@ rails server --binding=0.0.0.0
 You edit the code on your computer using the code editor you are used to using, then through shared folders the code is updated on the VM instantly.
 
 You should run the tests before submitting any patch or Pull Request back to the original repository. Please see [CONTRIBUTING.md](CONTRIBUTING.md) for more information.
+
+## Troubleshooting
+
+If you have issues installing ssh tunnel to the environment, try adding `:nfs_udp => false` to `override.vm.synced_folder` options in the `Vagrantfile`.
```

Comment:
I could not find a good box with xenial64 for it a couple weeks ago, hence this pull request.